tests/session/transaction-integration-001.phpt

```diff
@@ -0,0 +1,82 @@
+--TEST--
+MongoDB\Driver\Session::startTransaction() mini test
```

Comment:
Would "Committing a transaction" be a better name? I assume this doesn't encounter any errors and just demonstrates a transaction across two collections.

Reply:
I've done that. It's basically the same as the example from the PHPLIB requested docs: https://jira.mongodb.org/browse/PHPLIB-350